lib/kafo/logging.rb

```diff
@@ -46,7 +46,8 @@ def setup_file_logging(log_level, log_dir, log_owner, log_group)
             level: log_level,
             filename: filename,
             layout: layout(color: false),
-            truncate: true
```

Comment:
Every time I see diffs like this I'm convinced we should use more trailing commas.